macros/parsers/parserCheckboxList.pl

```diff
@@ -345,18 +364,27 @@ sub getCorrectChoices {
 		}
 
 		for (0 .. $#{ $self->{orderedChoices} }) {
-			if ($value eq $self->{orderedChoices}[$_] || $value eq ($self->{labels}[$_] || '')) {
+			if ($value eq $self->{orderedChoices}[$_]
+				|| $value eq ($self->{labels}[$_] || ''))
+			{
```

Comment:
I don't like this change either.  If a list of conditionals fits nicely into the 120 character limit it looks better on one line.